### DISCUSSION TITLE
I rewrote this project for use on docker-swarm

### DISCUSSION THREAD

**Discussion by freemo** [nginx-proxy/nginx-proxy#1959] [Show and tell]
Hi. I rewrote this project to work with docker-swarm, almost from scratch at this point but I did use this project as a starting point. Anyway I wanted to link it here for a few reasons 1) I'd like you to feel free to take any code you'd like from the project in order to make this project swarm-enabled, even take it all I dont need to maintain my own project. 2) So the authors can also comment or review if they wish.

Keep in mind I havent had a chance to look at the readmes or anything, I slapped this together quickly just a week or so ago and have been using it on my own swarm for about a week now. The repo I link below also contains a container that handles the letsencrypt side of things in much the same way nginx-proxy has.

https://git.qoto.org/modjular/swarm-proxy

**Comment by bluepuma77** [comment]
Really sad to see that Docker Swarm does not seem to have a bright future, despite being a great tool for 5-50 servers.

A similar ha-proxy project was abandoned two years ago: https://github.com/docker-flow/docker-flow-proxy 

Traefik has built-in Docker Swarm support, took me a while to get it up and running:
https://www.reddit.com/r/docker/comments/nf7dfo/how_to_setup_docker_swarm_traefik_24_domainbased/gyyhws1
